DiscordBot/commands/mud/move.js

```diff
@@ -75,6 +76,8 @@ class MoveCommand extends commando.Command {
         if (direction in room.exits) {
             // if a room exists in the given direction, use that direction's associated room ID to get the next room
             db.getItem(room.exits[direction], 'rooms', (data) => this.getRoom(message, data, direction, false));
+            console.log(room.exits[direction]);
+            this.client.channels.get(room.exits[direction]).send("Another player has entered.");
```

Comment:
would it be doable to change the message to `${message.member.nickname} has entered the area.`?

Reply:
Added. The player's name is now displayed in the next channel